package.json

```diff
@@ -28,16 +28,14 @@
   },
   "homepage": "https://github.com/serverless-heaven/serverless-webpack#readme",
   "scripts": {
-    "test": "mocha tests/all index.test.js \"lib/**/*.test.js\" -R spec --recursive",
-    "test-coverage": "nyc ./node_modules/mocha/bin/_mocha tests/all index.test.js \"lib/**/*.test.js\" -R spec --recursive",
-    "eslint": "eslint --ext .js *.js lib tests",
-    "format": "prettier-eslint \"$(pwd)/**/*.js\" --write"
```

Comment:
Does deleting `npm run format` means it is being replacing by `npm run eslint -- --fix` in the future?

Reply:
Well not in the future, it can already be using like that.
Do you want a `"eslint-fix": "eslint --ext .js *.js lib tests --fix",` script command?
And formatting will now make the `lint` build to fail too.

Reply:
> Do you want a "eslint-fix": "eslint --ext .js *.js lib tests --fix", script command?

No I think it's good enough already, just want to make sure I understand the change.